slc/component/matter-clusters/matter_level_control.slcc

```diff
@@ -9,8 +9,6 @@ quality: production
 metadata:
   sbom:
     license: "Apache 2.0"
-requires:
```

Comment:
Any reason this was removed?

Reply:
That's a manual change which is done after running the script. I have validated the pushed the change,